### PR TITLE
Add Mesos agent CSI configuration to install step.

### PR DESCRIPTION
**Pull request by greggomann**

### CODE REVIEW

File: frameworks/portworx/src/main/dist/svc.yml [hunk 1]
```diff
@@ -151,6 +151,19 @@ pods:
...
+
+                echo "{
+                    \"type\": \"pxd.portworx.com\",
+                    \"endpoints\": [
+                      {
+                        \"csi_service\": \"NODE_SERVICE\",
+                        \"endpoint\": \"unix:///var/lib/osd/driver/csi.sock\"
+                      }
+                    ],
+                    \"target_path_root\": \"/var/lib/osd/mounts/ucr\",
```
Comment (qianzhangxa) [inline frameworks/portworx/src/main/dist/svc.yml:163]: `ucr` is an internal component in Mesos, maybe here we should change it to `mesos`?

Comment (greggomann) [reply frameworks/portworx/src/main/dist/svc.yml:163]: UCR is more of a DC/OS-level marketing term that we use for the Mesos container runtime, so I think it's OK in this case. We use the term pretty widely in our docs.

**Review by disrani-px — APPROVED** [review]

**Comment by greggomann** [comment]
@disrani-px I was able to validate this PR in a DC/OS cluster; I created a volume using `pxctl`, then was able to launch tasks mounting it.

File: frameworks/portworx/universe/config.json [hunk 2]
```diff
@@ -68,7 +68,7 @@
           "portworx_image": {
             "title": "Portworx Image name",
             "type": "string",
-            "default": "portworx/px-enterprise:2.1.5"
+            "default": "portworx/px-enterprise:2.5.6"
```
Comment (disrani-px) [inline frameworks/portworx/universe/config.json:71]: The image also needs to be updated in universe/resource.json

**Review by disrani-px — APPROVED** [review]

**Pull request merged**